plugins/gatsby-theme-iterative-docs/index.js

```diff
@@ -0,0 +1 @@
+// noop
```

Comment:
Just something I noticed in other gatsby plugins/themes that seems to be standard practice if you're not using index.js.